exim/package/install.sls

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+# vim: ft=sls
+
+{#- Get the `tplroot` from `tpldir` #}
+{%- set tplroot = tpldir.split('/')[0] %}
+{%- from tplroot ~ "/map.jinja" import map with context %}
+
+exim/package/install:
```

Comment:
states ID are usually of the form `template-package-install-pkg-installed`, any reason to change it?

Reply:
I don't like it

Reply:
It's usually too verbose. Also depending on your salt output configuration it's nice to read.

Example: 

```sh
foo.bar.baz:
23:55:54.394906 [ 86.477 ms]    pkgrepo.managed    Clean     Name: telegraf/repo/install
23:56:00.766920 [ 33.916 ms]        pkg.installed  Clean     Name: telegraf/package/install
23:56:00.800984 [  13.08 ms]        pkg.installed  Clean     Name: telegraf/package/toml/install
23:56:00.816472 [118.376 ms]       file.managed    Clean     Name: telegraf/config/install
23:56:00.959043 [ 73.261 ms]    service.running    Clean     Name: telegraf/service/running
```

Reply:
And it's easier to parse (or it should be provided there's no file path in the state id)

Reply:
@noelmcloughlin The standardised state naming convention was implemented by you a while back.  While not a blocker for merging this PR, would you mind continuing this discussion?  Even if the PR is merged in the meantime?

Reply:
Actually my issue is the `-` as separator.

I clearly understand the intention and I'm totally ok with it :+1: 
But I think there's too many `-` and it doesn't help for reading.

The current state :

```sh
<formula>-<state>-<sub-state>-<action-id>
template-package-install-pkg-installed
```

The proposal :

```sh
<formula>/<state>/<sub-state>/<action-id>
template/package/install/pkg-installed
or the less redundant
template/package/install
```

Reply:
@n-rodriguez The one issue with forward-slashes is that they can easily be confused with file paths, which are used for state names as well.  My (selfish) preference is for underscores, since that grabs the whole state name when yanking in Vim!  Otherwise, the other angle is ask @noelmcloughlin to explain all of the components of the state name (and why it ends up so long).  There was some talk of an API but I can't remember the details.

Reply:
> The one issue with forward-slashes is that they can easily be confused with file paths, which are used for state names as well.

With this "convention" (forward-slashes as separator) the state name **is** a path (or become path) : the path to the sls file (or to the salt action).

Reply:
@n-rodriguez Thanks for the explanation.  Another question for you: how about the `<action-id>` component?  What if there are multiple states in one file, what would be the convention to distinguish between them?

Reply:
See https://github.com/saltstack-formulas/apt-cacher-formula/pull/19/commits/d1b56c232a086ec0382a431567ee75949c55eff0

Reply:
@n-rodriguez Right, so essentially the same thing, using forward-slashes instead of a dashes.  The current method is more formulaic but your method is more flexible and meaningful.  I can see the benefits of both.  Hopefully, others will have some input.  @noelmcloughlin, I'd love to hear your take on this discussion.